### DISCUSSION TITLE
Windows version based on UWP platform

### DISCUSSION THREAD

**Discussion by cryptedjeffrey** [f1multiviewer/issue-tracker#329] [Feature requests] [answered]
Hi,

It would be awesome to see this project as an UWP project, meaning that it would (likely) become Xbox compatible, generating a whole new user base for it.

Keep up the good work!

**Comment by f1multiviewer** [comment, marked as answer]
Unfortunately that would be impossible in various ways (Widevine support lacking, this project being Electron, and the small market share of Xbox-users compared to other devices). This will be *very* unlikely to ever be done by MultiViewer
